### ISSUE TITLE
Variables side panel doesn't load on version 32.10 query editor

### ISSUE THREAD

**Issue by adriank-convoy** [metabase/metabase#10524]
**Describe the bug**
When editing a native query, the variables side panel does not appear when a `{{variable}}` is entered into the query box, so it's not possible to setup/use variables.

**Logs**

Logs from docker container (nothing logged during query creation, no GET requests or anything... these are the logs from executing the query)
```
08-07 21:31:06 WARN middleware.process-userland-query :: Query failure {:status :failed,
 :class clojure.lang.ExceptionInfo,
 :error "Unable to substitute 'date_part': param not specified.\nFound: ()",
 :stacktrace
 ("--> util.i18n$ex_info.invokeStatic(i18n.clj:103)"
  "util.i18n$ex_info.invoke(i18n.clj:99)"
  "query_processor.middleware.parameters.sql$fn__39305$parse_params_or_throw__39310$fn__39311.invoke(sql.clj:534)"
  "query_processor.middleware.parameters.sql$fn__39305$parse_params_or_throw__39310.invoke(sql.clj:529)"
  "query_processor.middleware.parameters.sql$fn__39334$parse_optional_params__39339$fn__39340.invoke(sql.clj:557)"
  "query_processor.middleware.parameters.sql$fn__39334$parse_optional_params__39339.invoke(sql.clj:551)"
  "query_processor.middleware.parameters.sql$fn__39373$parse_template__39378$fn__39379.invoke(sql.clj:565)"
  "query_processor.middleware.parameters.sql$fn__39373$parse_template__39378.invoke(sql.clj:562)"
  "query_processor.middleware.parameters.sql$fn__39397$expand_query_params__39402$fn__39406.invoke(sql.clj:575)"
  "query_processor.middleware.parameters.sql$fn__39397$expand_query_params__39402.invoke(sql.clj:573)"
  "query_processor.middleware.parameters.sql$expand.invokeStatic(sql.clj:581)"
  "query_processor.middleware.parameters.sql$expand.invoke(sql.clj:577)"
  "query_processor.middleware.parameters$expand_parameters_STAR_.invokeStatic(parameters.clj:21)"
  "query_processor.middleware.parameters$expand_parameters_STAR_.invoke(parameters.clj:14)"
  "query_processor.middleware.parameters$expand_parameters.invokeStatic(parameters.clj:44)"
  "query_processor.middleware.parameters$expand_parameters.invoke(parameters.clj:40)"
  "query_processor.middleware.parameters$substitute_parameters_STAR_.invokeStatic(parameters.clj:50)"
  "query_processor.middleware.parameters$substitute_parameters_STAR_.invoke(parameters.clj:47)"
  "query_processor.middleware.driver_specific$process_query_in_context$fn__35805.invoke(driver_specific.clj:12)"
  "query_processor.middleware.resolve_driver$resolve_driver$fn__39746.invoke(resolve_driver.clj:23)"
  "query_processor.middleware.bind_effective_timezone$bind_effective_timezone$fn__34996$fn__34997.invoke(bind_effective_timezone.clj:9)"
  "util.date$call_with_effective_timezone.invokeStatic(date.clj:88)"
  "util.date$call_with_effective_timezone.invoke(date.clj:77)"
  "query_processor.middleware.bind_effective_timezone$bind_effective_timezone$fn__34996.invoke(bind_effective_timezone.clj:8)"
  "query_processor.middleware.store$initialize_store$fn__42870$fn__42871.invoke(store.clj:11)"
  "query_processor.store$do_with_store.invokeStatic(store.clj:46)"
  "query_processor.store$do_with_store.invoke(store.clj:40)"
  "query_processor.middleware.store$initialize_store$fn__42870.invoke(store.clj:10)"
  "query_processor.middleware.async$async__GT_sync$fn__19259.invoke(async.clj:23)"
  "query_processor.middleware.async_wait$runnable$fn__34724.invoke(async_wait.clj:89)"),
 :query
 {:type "native",
  :native {:query "SELECT DATE_TRUNC({{date_part}}, current_timestamp())", :template-tags {}},
  :parameters [],
  :async? true,
  :middleware {:add-default-userland-constraints? true, :userland-query? true},
  :info
  {:executed-by 1,
   :context :ad-hoc,
   :card-id nil,
   :nested? false,
   :query-hash [-4, 114, -49, 83, -66, -6, -102, -112, 92, 52, -126, -5, 0, 2, -96, -32, 22, -54, -95, -8, 103, -27, -61, 62, 2, -82, 45, -31, -14, -37, -106, 88]},
  :constraints {:max-results 10000, :max-results-bare-rows 2000}},
 :ex-data {:status-code 400}}
```
One error in the JS console:

```
app-main.bundle.js?db89dea08bd003ad80f4:5 Uncaught TypeError: Cannot read property 'offsetHeight' of null
    at t.value (app-main.bundle.js?db89dea08bd003ad80f4:5)
    at t.value (app-main.bundle.js?db89dea08bd003ad80f4:5)
    at u (vendor.bundle.js?db89dea08bd003ad80f4:1)
value @ app-main.bundle.js?db89dea08bd003ad80f4:5
value @ app-main.bundle.js?db89dea08bd003ad80f4:5
u @ vendor.bundle.js?db89dea08bd003ad80f4:1
setTimeout (async)
(anonymous) @ vendor.bundle.js?db89dea08bd003ad80f4:1
r._signal @ vendor.bundle.js?db89dea08bd003ad80f4:1
onChange @ vendor.bundle.js?db89dea08bd003ad80f4:1
r._signal @ vendor.bundle.js?db89dea08bd003ad80f4:1
applyDelta @ vendor.bundle.js?db89dea08bd003ad80f4:1
insertMergedLines @ vendor.bundle.js?db89dea08bd003ad80f4:1
insert @ vendor.bundle.js?db89dea08bd003ad80f4:1
setValue @ vendor.bundle.js?db89dea08bd003ad80f4:1
setValue @ vendor.bundle.js?db89dea08bd003ad80f4:1
value @ app-main.bundle.js?db89dea08bd003ad80f4:5
value @ app-main.bundle.js?db89dea08bd003ad80f4:5
t.notifyAll @ vendor.bundle.js?db89dea08bd003ad80f4:1
close @ vendor.bundle.js?db89dea08bd003ad80f4:1
closeAll @ vendor.bundle.js?db89dea08bd003ad80f4:1
perform @ vendor.bundle.js?db89dea08bd003ad80f4:1
perform @ vendor.bundle.js?db89dea08bd003ad80f4:1
perform @ vendor.bundle.js?db89dea08bd003ad80f4:1
w @ vendor.bundle.js?db89dea08bd003ad80f4:1
closeAll @ vendor.bundle.js?db89dea08bd003ad80f4:1
perform @ vendor.bundle.js?db89dea08bd003ad80f4:1
batchedUpdates @ vendor.bundle.js?db89dea08bd003ad80f4:1
batchedUpdates @ vendor.bundle.js?db89dea08bd003ad80f4:1
dispatchEvent @ vendor.bundle.js?db89dea08bd003ad80f4:1
```

**To Reproduce**
1. Start a 32.10 docker container `docker run -d -p 3000:3000 --name metabase metabase/metabase`
2. Create a new native query
3. Enter `SELECT DATE_TRUNC({{date_part}}, current_timestamp())`

**Expected behavior**
Variable side panel loads for `date_part`  (image from older Metabase version, I confirmed this works in the 32.9 docker image)

![image](https://user-images.githubusercontent.com/40185566/62659610-b3507f80-b920-11e9-9e6f-f627c761ea51.png)

**Actual behavior**

![image](https://user-images.githubusercontent.com/40185566/62659635-c1060500-b920-11e9-952d-7751a978b0dd.png)

**Information about your Metabase Installation:**
- Your browser and the version: Version 75.0.3770.142 (Official Build) (64-bit)

- Your operating system: OS X

- Your databases: Snowflake

- Metabase version: Official 32.10 docker image

- Metabase hosting environment: Docker

- Metabase internal database: H2 

**Severity**

P1, we can't upgrade/use this version until this is fixed

**Comment by tlrobinson** [comment]
FYI this appears to be fixed in `release-0.33.x`, but maybe we should fix in `v0.32.11`?

**Comment by tlrobinson** [comment]
I'm also unable to reproduce this on `v0.32.10` using Postgres or H2 databases. It's possible it's specific to Snowflake.

Has anyone else reproduced it?

**Comment by tlrobinson** [comment]
I was unable to reproduce this using Snowflake as well. Chrome 76.0.3809.100.

@adriank-convoy Does this happen consistently? Any other notable things about your setup?

**Comment by flamber** [comment]
I don't have Snowflake, but I cannot reproduce on 0.32.10 with many different setups.

One thing I noticed in the two screenshots. The database name is different. `Snowflake` and `SNowflake`.
Is that different databases or is it because it's different H2 metadata?

**Comment by adriank-convoy** [comment]
Same Snowflake instance, different H2 metadata (I tested with two docker containers - one of our current version 0.31.2, and one with 0.32.10). I guess it may be something with my Chrome setup? I'll try reproducing on another machine sometime today.

**Comment by tlrobinson** [comment]
@adriank-convoy Just checking if you were able to reproduce this, or if it was resolved in the 0.33.x releases?

**Comment by adriank-convoy** [comment]
Sorry haven't had time to try, let's just close this and I can re-open if I can reproduce it on 0.33.x
